pkg/api/v1alpha2/capsule_types.go

```diff
@@ -227,6 +227,50 @@ type CapsuleInterface struct {
 
 	// Public specifies if and how the interface should be published.
 	Public *CapsulePublicInterface `json:"public,omitempty"`
+
+	// Host routes that are mapped to this interface.
+	Routes []HostRoute `json:"routes,omitempty"`
+}
+
+// HostRoute is the configuration of a route to the network interface
+// it's configured on.
+type HostRoute struct {
+	// Host of the route. The value "*" means a wildcard, that matches any hostname.
+	// The character "*" is not valid in any other cases.
```

Comment:
I cant think of a real use case where you would actually want routing not based on a host. Maybe we should just require a host?

Reply:
Agreed!